src/js/components/Button/StyledButton.js

```diff
@@ -11,13 +11,20 @@ import {
 } from '../../utils';
 import { defaultProps } from '../../default-props';
 
+const radiusStyle = props => {
+  if (props.size) {
+    return props.theme.button.size[props.size].border.radius;
```

Comment:
If there isn't a size in the theme, I think we should fall back to `button.border.radius`.

Reply:
Good catch. done.